### PR TITLE
LPS-133417 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133417

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 1]
```diff
@@ -63,6 +63,50 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-125388. The new user can edit the Home page with site member permission."
+	@priority = "4"
+	test CanEditHomePageWithSiteMemberPermission {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:68]: EditSpecificContentPageAsSiteMemberWithUpdatePermission

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 2]
```diff
@@ -63,6 +63,50 @@ definition {
...
+		task ("Assign the user to site") {
+			JSONUser.addUserToSite(
+				groupName = "Guest",
+				userEmailAddress = "userea@liferay.com");
+		}
+
+		task ("Grant permission Update to the site member role") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:75]: Grant Update permission of specific content page to Site Member

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 3]
```diff
@@ -63,6 +63,50 @@ definition {
...
+			SitePages.openPagesAdmin(siteURLKey = "Guest");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:76]: Test Site Name

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 4]
```diff
@@ -63,6 +63,50 @@ definition {
...
+
+			SitePages.gotoPageMenuItem(
+				menuItem = "Permissions",
+				pageName = "Home");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:80]: Test Content Page Name

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 5]
```diff
@@ -63,6 +63,50 @@ definition {
...
+
+			IFrame.selectPermissionsFrame();
+
+			Check(
+				key_actionName = "UPDATE",
+				key_roleTitle = "site-member",
+				locator1 = "Permissions#PERMISSIONS_MODAL_CHECKBOX");
+
+			Button.clickSave();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:89]: PortletEntry.save();

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you update the case according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you update the case according to comments? Thanks

Hi @Tim-Cao, PR Updated.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 6]
```diff
@@ -787,6 +787,45 @@ definition {
...
+		}
+
+		task ("Sign in as new user") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+		}
+
+		task ("Add a Paragraph fragment") {
+			ContentPagesNavigator.openEditContentPage(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:817]: Should use the Click pencil icon at view mode

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 7]
```diff
@@ -787,6 +787,45 @@ definition {
...
+				pageName = "Test Content Page Name",
+				siteName = "Test Site Name");
+
+			PageEditor.addFragment(
+				collectionName = "Basic Components",
+				fragmentName = "Paragraph");
+
+			PageEditor.clickPublish();
+		}
+	}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:827]: Add step to view Paragraph at view mode

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/af721ac926478f63e74ea7783b08b147d3a2ad0c...fbdaacc5e13ca6d979c76d96c0370aa2c36f8526

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**